lms/djangoapps/verify_student/views.py

```diff
@@ -1004,10 +1026,15 @@ def results_callback(request):
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        _set_requirement_status(attempt, 'reverification', 'satisfied')
```

Comment:
`status` is defined above, but you don't use it when calling `_set_requirement_status`, here or below. As a result, the system will use inconsistent status messages when, for example, setting the requirement status and when composing reverification emails. Is this intended?